Header header
string neo_cmd                # 'TRACK_ROI'/'TALK_GESTURE'/'HAND_OP'/'LEG_OP'/  : high level cmd.
---
uint8  result


